domain/consensus/processes/blockbuilder/block_builder.go

```diff
@@ -145,6 +145,11 @@ func (bb *blockBuilder) validateTransaction(
 		return err
 	}
 
+	err = bb.transactionValidator.ValidateTransactionWithContextAndWithoutUTXOContext(stagingArea, transaction, model.VirtualBlockHash)
```

Comment:
ValidateTransactionInContextAndPopulateMassAndFee already calls this. Why call it twice?

Reply:
I can't find where, can you show me maybe?